src/helpers/logger.tsx

```diff
@@ -0,0 +1,7 @@
+const logger = {
+    log: (message?: any, ...optionalParams: any[]) => {
+        console.log(message, ...optionalParams);
+    },
+};
+
+export default logger;
```

Comment:
Kunne det ikke være grinern at exportere functionen, sådan her:

```suggestion
export const log = (message?: any, ...optionalParams: any[]) => {
    console.log(message, ...optionalParams);
};
```

Så kan vi måske lave en error senere eller sådan noget, om den måde du importere den på, er så ved at skrive
import { log, error } from 'logger'

log("hej");
error("Fucking skrald pis i9 lortet lort");